installation/aws/VmClarity.cfn

```diff
@@ -322,42 +342,80 @@ Resources:
                     ExecStartPre=/usr/bin/docker pull ${FreshclamMirrorContainerImage}
                     ExecStart=/usr/bin/docker run \
                       --rm --name %n \
+                      --network vmclarity \
                       -p 0.0.0.0:1000:80/tcp \
                       ${FreshclamMirrorContainerImage}
 
                     [Install]
                     WantedBy=multi-user.target
                   - FreshclamMirrorContainerImage: !If [FreshclamMirrorContainerImageOverridden, !Ref FreshclamMirrorContainerImageOverride, "ghcr.io/openclarity/freshclam-mirror:v0.1.0"]
               mode: "000644"
+            "/lib/systemd/system/postgres.service":
+              content:
+                Fn::Sub:
+                  - |
+                    [Unit]
+                    Description=Postgres Database Server
+                    After=docker.service
+                    Requires=docker.service
+
+                    [Service]
+                    TimeoutStartSec=0
+                    Restart=always
+                    ExecStartPre=-/usr/bin/docker stop %n
+                    ExecStartPre=-/usr/bin/docker rm %n
+                    ExecStartPre=/usr/bin/docker pull ${PostgresContainerImage}
+                    ExecStart=/usr/bin/docker run \
+                      --rm --name %n \
+                      --network vmclarity \
+                      -e POSTGRESQL_USERNAME=vmclarity \
+                      -e POSTGRESQL_PASSWORD=password123 \
+                      -e POSTGRESQL_DATABASE=vmclarity \
+                      -p 127.0.0.1:5432:5432/tcp \
+                      ${PostgresContainerImage}
+
+                    [Install]
+                    WantedBy=multi-user.target
+                  - PostgresContainerImage: "bitnami/postgresql:12.14.0-debian-11-r28" # !If [PostgresContainerImageOverridden, !Ref PostgresContainerImageOverride, "ghcr.io/openclarity/freshclam-mirror:v0.1.0"]
```

Comment:
ghcr.io/openclarity/freshclam-mirror:v0.1.0 ... ?

Reply:
That part of the yaml is commented out, I've left it there as an example for me to add the override logic for the postgres container